### DISCUSSION TITLE
9주차 Discussion

### DISCUSSION THREAD

**Discussion by chloe-codes1** [slippStudy/java-network-study#16] [General]
## 📢 책을 읽으면서 궁금한 점들을 자유롭게 적고 대답해 주세요

9주차 진행 방식

> 진행자: 동훈님
> 

- 16장 읽기
- Discussion 에 질문 or 인상 깊은 점 2개 적어오기

**Comment by jinwookh** [comment]
1. 실제 프레임워크에서는 셀렉터를 몇 개 사용할까?
2. 예제에서 빠져있는 부분이 있었는데, 채워넣는 과정이 재밌었다.

**Comment by kangbada0728** [comment]
네트워크에 대해서 더 깊이 알 수 있는 기회였습니다. 정말 좋았어요. 같이 공부하길 잘했습니다.

**Comment by chloe-codes1** [comment]
Java IO / NIO package 를 깊게 들여다 볼 수 있어 좋았습니다!
고생하셨습니다~!

**Comment by LenKIM** [comment]
개인적으로 코드 패턴에 대한 의구심이 들었다.
ObjectPoolPattern 이나, ServiceLoader Pattern, Adapter 패턴 등이 사용된 것이 인상깊었다.

**Comment by yangyunhyeuk** [comment]
1. 예제 코드 내용 중에서 직접적 상속을 받는 방식이 아닌, 핸들러어댑터를 생성하여 추상클래스를 상속받는 구조가 인상깊었습니다.
